src/Rx/Weno/PharmaciesController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ *  @package   OpenEMR
+ *  @link      http://www.open-emr.org
+ *  @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c )2019. Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ *
+ */
+
+namespace OpenEMR\Rx\Weno;
```

Comment:
place a empty line to separate namespace block from use block

Reply:
Done